rasa/shared/data.py

```diff
@@ -172,3 +172,21 @@ def _copy_files_to_new_dir(files: Iterable[Text]) -> Text:
         shutil.copy2(f, os.path.join(directory, unique_file_name))
 
     return directory
+
+
+class TrainingType(Enum):
+    """Enum class for defining explicitly what training types exist."""
+
+    NLU = 1
+    CORE = 2
+    BOTH = 3
+    END_TO_END = 4
+
+    @property
+    def model_type(self) -> Text:
+        """Returns the type of model which this training yields."""
+        if self == TrainingType.NLU:
+            return "nlu"
+        if self == TrainingType.CORE:
+            return "core"
+        return "rasa"
```

Comment:
Good catch 😅  In this case I moved this Enum class from importer module since recipes also need to know about training type and I didn't change it. `model_type` is not used in many places (mostly for telemetry) and I guess the only relevant values when used are NLU and core at the moment.

Reply:
Oh forgot the part about file, I checked constants.py but it seemed only to have single line data types such as strings and this being a class with property method (model_type) I thought it made more sense put it here. I don't have strong opinions though, and I can move it there @ka-bu

Reply:
If it had more functionality attached, then I think this would be a good place but since it's just this, I'd move it to the constants (that don't specify that only string constants are allowed). -- If no one else is confused, then feel free to leave it here